### PR TITLE
Adds Support for chaining classnames

### LINKED ISSUE #2: Doesn't properly support Composition & Dependencies
When composing a class, the composed classname is added to the JSON mapping. Because it's one string, the helper functions aren't properly formatting the return values.

I noticed this just now with `getComponentSelectors`, but I'm sure the others are affected by this oversight as well.

Given the following:

```css
/* theme.css */
.btn {
  border: 1px solid;
  color: red;
}
```

```css
/* header.css */
.wrapper {
  composes: btn from 'theme.css';
}
```

```json
{"header":{"wrapper":"header__wrapper___2JY86 theme__btn___2TPiI"}}
```

The output from `getComponentSelectors`

```javascript
const selectors = getComponentSelectors('header');
// '.header__wrapper___2JY86 theme__btn___2TPiI'
```

### PR DESCRIPTION
**Pull request by goodguyry**
Fixes #2

### CODE REVIEW

File: src/util/getChainedSelector.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Combine a string of classnames into a chained selector.
+ *
+ * @param  {string} selectors The selector list.
+ * @return {string}           The merged, formatted selectors.
+ */
+export default function getChainedSelector(selectors) {
+  const classList = selectors.split(' ').map((classname) => `.${classname}`);
+  return classList.join('');
```
Comment (kevinfodness) [inline src/util/getChainedSelector.js:9]: Could be one-lined-ish:
```suggestion
return selectors.split(' ')
  .map((classname) => `.${classname}`)
  .join('');
```

File: src/util/getChainedSelector.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable max-len */
+import getChainedSelector from './getChainedSelector';
+
+describe('Merges a string of classnames and formats them as a compound selector', () => {
+  test.each([
+    ['selector-one selector-two', '.selector-one.selector-two'],
+    ['selector-one selector-two menu-item-link', '.selector-one.selector-two.menu-item-link'],
+    ['menu-item-link', '.menu-item-link'],
```
Comment (kevinfodness) [inline src/util/getChainedSelector.spec.js:8]: Should add a test for an empty string to ensure it returns an empty string.

Comment (goodguyry) [reply src/util/getChainedSelector.spec.js:8]: Good catch 🙌
